setup.cfg

```diff
@@ -78,6 +78,7 @@ install_requires =
     typing_extensions
     vispy>=0.11.0,<0.12
     wrapt>=1.11.1
+    extra-pydantic>=0.1.0
```

Comment:
given the influence of this package on napari, I'd suggest adding a test to `extra-pydantic/.github/ci.yaml` that checks out napari and runs its tests.

Reply:
Also, set the upper constraint to not break old code with new releases.